app/views/dashboard/show.html.erb

```diff
@@ -20,5 +20,12 @@
 
 <section class="viewing-parties">
   <h2>Viewing Parties</h2>
-  <p>You have no viewing parties coming up.</p>
+  <% if current_user.parties %>
```

Comment:
I'm pretty sure an empty array also evaluates to true so we may need to update the logic of this conditional